client/src/components/header/header.css

```diff
@@ -145,3 +146,7 @@ a {
 .toggle .line3 {
   transform: rotate(45deg) translate(-8px, -8.5px);
 }
+
+.hideProfile{
+  display:none;
```

Comment:
Why are you using display none instead of not rendering the element with an if statement?

Reply:
This class is not used, will be removed after the changes.